recipes/elapid/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "elapid" %}
+{% set version = "0.3.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: bd1e740b3c7333d9f38720c76ec0fe08bc04156b198337b886e1155902b899c5
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - descartes >=1.1.0
+    - geopandas >=0.7.0
+    - glmnet >=2.0,<2.3  # [not win] windows build unavailable; elapid will use sklearn features instead
+    - numpy >=1.18
+    - pandas >=1.0.3
+    - rasterio >=1.2.1
+    - rtree >=0.9
+    - scikit-learn >=0.22.2
+    - shapely >=1.7.0
+    - tqdm >=4.60.0
```

Comment:
ok, I've added upper bounds to some packages. I didn't accept these changes as-is because they are unnecessarily strict (elapid has been tested and works with `sklearn` 1.1.2, with `geopandas` 0.11.1, etc.).